packages/typespec-client-generator-core/test/types/duration-type.test.ts

```diff
@@ -115,8 +115,10 @@ describe("typespec-client-generator-core: duration types", () => {
     strictEqual(sdkType.kind, "array");
     strictEqual(sdkType.valueType.kind, "duration");
     strictEqual(sdkType.valueType.name, "Float32Duration");
-    strictEqual(sdkType.valueType.description, "title");
-    strictEqual(sdkType.valueType.details, "doc");
+    strictEqual(sdkType.valueType.description, "title"); // eslint-disable-line deprecation/deprecation
```

Comment:
Can we test with different locales like Chinese etc?

Reply:
is it kind of test for typespec, not for tcgc?

Reply:
added.